core/domain/suggestion_services_test.py

```diff
@@ -1737,6 +1761,42 @@ def setUp(self):
             self.REVIEWER_EMAIL)
         self.save_new_valid_exploration(self.target_id, self.author_id)
 
+    def test_create_raises_for_suggestion_type_not_on_contributor_dashboard(
```

Comment:
Is it possible for us to use a synthetic type in case edit state content is someday shown on the contributor dashboard, or will it never be? Otherwise this test will start incorrectly failing, and it may be hard to replace it with a correct alternative (especially if all suggestion types eventually end up on the contributor dashboard).

Reply:
This is a good point but it's not possible to create a synthetic suggestion type.

Reply:
Could we use a swap to make sure that the types being checked against here never include edit state content as a valid type? That's not a very robust way to test this case, but it's at least a reasonable proxy for that check.

Reply:
Changed

Reply:
Thanks for updating the test!